students/ReemAlqaysi/Lesson01/Lesson01_Exceptions.py

```diff
@@ -0,0 +1,49 @@
+
+#By Reem Alqaysi
+#lesson01
+""" script should calculate Lunch portion size for a reguler day, but it's Break me code """
+
+#This Function will cause NameError exception
+def lunch_NameError():
```

Comment:
Make sure function names (and most other things except classes) are snake_case in Python.  So, def lunch_name_error():  Even though it's giving an example of a NameError.